table_test.go

```diff
@@ -259,7 +259,7 @@ func TestTryAddPeer(t *testing.T) {
 
 	// adding a peer with cpl 0 works if an existing peer has LastSuccessfulOutboundQuery above the max threshold
 	// because that existing peer will get replaced
-	require.True(t, rt.UpdateLastSuccessfulOutboundQuery(p2, time.Now().AddDate(0, 0, -1)))
+	require.True(t, rt.UpdateLastSuccessfulOutboundQuery(p2, time.Now().AddDate(0, 0, -2)))
 	b, err = rt.TryAddPeer(p3, true)
 	require.NoError(t, err)
 	require.True(t, b)
```

Comment:
I can make this more clear in the code, but the issue is basically that the threshold is 1 day `minThreshold := float64(24 * 1 * time.Hour)` which means that the internal comparison of if the peer has not been updated recently may fail unless the time between line 262 and the check in line 263 takes enough time. By setting the time difference to two days instead of one that's not going to end up happening unless your test is REALLLLYYY slow 😄.

Not sure if others have experienced the failure or if it has to do with me running the tests on Windows and dealing with Go not having nanosecond time on Windows.